sdk/src/process_instruction.rs

```diff
@@ -321,6 +322,24 @@ pub mod stable_log {
         ic_logger_msg!(logger, "Program log: {}", message);
     }
 
+    /// Log event data as from the program itself.
+    ///
+    /// The general form is:
+    ///
+    /// ```notrust
+    /// "Program event data: <program-id> <event-data-in-base64>+"
+    /// ```
+    ///
+    /// That is, any program-generated output is guaranteed to be prefixed by "Program event data: "
+    pub fn program_event(logger: &Rc<RefCell<dyn Logger>>, program_id: &Pubkey, data: &[&[u8]]) {
+        ic_logger_msg!(
+            logger,
+            "Program return data: {} {}",
```

Comment:
```suggestion
            "Program event: {} {}",
```

Reply:
wdyt about shortening "Program return data:" to "Program return:" too?

Reply:
Good idea